### DISCUSSION TITLE
Password hash containing "argon2" is that intentional?

### DISCUSSION THREAD

**Discussion by yarapolana** [adonisjs/core#1660] [Help] [answered]
I was trying out creating a user and in the database I noticed the password comes out like this:

<img width="766" alt="Screenshot 2020-09-25 at 00 01 43" src="https://user-images.githubusercontent.com/19730118/94211991-777cce80-fec2-11ea-8d66-aa70659676ff.png">

In v4.1 I think it was justt the token, is that intentional to have `$argon2...`

**Comment by thetutlage** [comment, marked as answer]
We follow the PHC hashing format https://github.com/P-H-C/phc-string-format/blob/master/phc-sf-spec.md and adding `$argon2` is part of the standard. 

The main goal of the standard is to allow upgrading old password hash when you change the config at a later stage and hence it appends some meta-data to each password

**Reply by yarapolana** [reply]
Got it. Thanks.
